hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/cdc/TestCDCDataFrameSuite.scala

```diff
@@ -118,6 +118,7 @@ class TestCDCDataFrameSuite extends HoodieCDCTestBase {
       .option(DataSourceWriteOptions.OPERATION.key, DataSourceWriteOptions.DELETE_OPERATION_OPT_VAL)
       .option("hoodie.clustering.inline", "true")
       .option("hoodie.clustering.inline.max.commits", "1")
+      .option("hoodie.clustering.plan.strategy.sort.columns", "_row_key")
       .mode(SaveMode.Append)
```

Comment:
Without this change，after inputDF5 the timeline will be:
   commit(instantC,cleaned)->clustering(instantD,cleaned)->commit(instantE)->clean(instantF)
and file belong to instantA will archived,  it will case line-193 -> line 194:
-------------------------------------------------------------------------------------------
   allVisibleCDCData = cdcDataFrame((commitTime1.toLong - 1).toString)
   assertCDCOpCnt(allVisibleCDCData, totalInsertedCnt, totalUpdatedCnt, totalDeletedCnt)
-------------------------------------------------------------------------------------------
can't get instantA file error. 

With this change, after after inputDF5 the timeline will be:
   clustering(instantD,cleaned)->clustering(instantE,cleaned)->commit(instantF)->clean(instantG),
it will fix this problem.

Reply:
Without this change，after inputDF5 the timeline will be:
   commit(instantC,cleaned)->clustering(instantD,cleaned)->commit(instantE)->clean(instantF)
and file belong to instantA will archived,  it will case line-193 -> line 194:
   allVisibleCDCData = cdcDataFrame((commitTime1.toLong - 1).toString)
   assertCDCOpCnt(allVisibleCDCData, totalInsertedCnt, totalUpdatedCnt, totalDeletedCnt)
can't get instantA file error. 

With this change, after after inputDF5 the timeline will be:
   clustering(instantD,cleaned)->clustering(instantE,cleaned)->commit(instantF)->clean(instantG),
it will fix this problem.

Reply:
Okey, seems you wanna keep the clustering for single filegroup by adding sort columns.